src/bundle/Resources/public/js/scripts/helpers/system.helper.js

```diff
@@ -1,79 +1,77 @@
+    }
 
-        if (isWindows() || isLinux()) {
-            return event.ctrlKey && event.key === letter;
-        }
+    if (isWindows() || isLinux()) {
+        return event.ctrlKey && event.key === letter;
+    }
 
-        return false;
-    };
-    const isUndoPressed = (event) => {
-        if (isMac()) {
-            return event.metaKey && !event.shiftKey && event.key === 'z';
-        }
+    return false;
+};
+const isUndoPressed = (event) => {
+    if (isMac()) {
+        return event.metaKey && !event.shiftKey && event.key === 'z';
+    }
 
-        if (isWindows() || isLinux()) {
-            return event.ctrlKey && event.key === 'z';
-        }
+    if (isWindows() || isLinux()) {
+        return event.ctrlKey && event.key === 'z';
+    }
 
-        return false;
-    };
+    return false;
+};
 
-    const isRedoPressed = (event) => {
-        if (isMac()) {
-            return event.metaKey && event.shiftKey && event.key === 'z';
-        }
+const isRedoPressed = (event) => {
+    if (isMac()) {
+        return event.metaKey && event.shiftKey && event.key === 'z';
+    }
 
-        if (isWindows() || isLinux()) {
-            return event.ctrlKey && event.key === 'y';
-        }
+    if (isWindows() || isLinux()) {
+        return event.ctrlKey && event.key === 'y';
+    }
 
-        return false;
-    };
-    const isSavePressed = (event) => {
-        return isShortcutWithLetter(event, 's');
-    };
-    const isCopyPressed = (event) => {
-        return isShortcutWithLetter(event, 'c');
-    };
-    const isCutPressed = (event) => {
-        return isShortcutWithLetter(event, 'x');
-    };
-    const isPastePressed = (event) => {
-        return isShortcutWithLetter(event, 'v');
-    };
-    const isPrintPressed = (event) => {
-        return isShortcutWithLetter(event, 'p');
-    };
-    const isSelectAllPressed = (event) => {
-        return isShortcutWithLetter(event, 'a');
-    };
+    return false;
+};
+const isSavePressed = (event) => {
+    return isShortcutWithLetter(event, 's');
+};
+const isCopyPressed = (event) => {
+    return isShortcutWithLetter(event, 'c');
+};
+const isCutPressed = (event) => {
+    return isShortcutWithLetter(event, 'x');
+};
+const isPastePressed = (event) => {
+    return isShortcutWithLetter(event, 'v');
+};
+const isPrintPressed = (event) => {
+    return isShortcutWithLetter(event, 'p');
+};
+const isSelectAllPressed = (event) => {
+    return isShortcutWithLetter(event, 'a');
+};
 
-    ibexa.addConfig('helpers.system', {
-        isWindows: isWindows(),
-        isMac: isMac(),
-        isLinux: isLinux(),
-        isUndoPressed,
-        isRedoPressed,
-        isSavePressed,
-        isCopyPressed,
-        isCutPressed,
-        isPastePressed,
-        isPrintPressed,
-        isSelectAllPressed,
-        isShortcutWithLetter,
-    });
-})(window, window.document, window.ibexa);
+export {
+    isWindows,
+    isMac,
+    isLinux,
```

Comment:
```suggestion
    get isWindows() { return isWindows(); }
```
@lucasOsti, @GrabowskiM, @tischsoic, @dew326 
What do you think about this approach?
I can add it in seperate PR.
So we can get property like helper.isWindows instead of helper.isWindows() when we only return bool true or false

Reply:
Current upside of using `../../..` is that VS Code understands where a file is. If you use an alias, this file won't be recognized